sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -55,4 +56,49 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
     } as InstitutionLocation;
     return this.repo.save(programOffering);
   }
+
+  /**
+   * This is to fetch all the Education Offering
+   * that are associated with the Location and Program
+   * @param locationId
+   * @param programId
+   * @returns
+   */
+  async getAllEducationProgramOffering(
+    locationId: number,
+    programId: number,
+  ): Promise<EducationProgramOfferingModel[]> {
+    const educationProgramOfferingResult = await this.repo
+      .createQueryBuilder("education_programs_offerings")
+      .select([
+        "education_programs_offerings.id",
+        "education_programs_offerings.name",
+        "education_programs_offerings.studyStartDate",
+        "education_programs_offerings.studyEndDate",
+        "education_programs_offerings.offeringDelivered",
+      ])
+      .leftJoin(
```

Comment:
I think that this could be a join instead of a leftJoin.

Reply:
done